openshift-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/OpenshiftHelmPushMojo.java

```diff
@@ -25,22 +26,15 @@
 public class OpenshiftHelmPushMojo extends HelmPushMojo {
 
   /**
-   * The generated kubernetes YAML file
-   */
-  @Parameter(property = "jkube.kubernetesManifest", defaultValue = "${basedir}/target/classes/META-INF/jkube/openshift.yml")
-  private File openShiftManifest;
-
-  /**
-   * The generated kubernetes YAML file
+   * One of:
+   * <ul>
+   * <li>A directory containing OpenShift Templates to use as Helm parameters.</li>
+   * <li>A file containing a Kubernetes List with OpenShift Template entries to be used as Helm parameters.</li>
+   * </ul>
    */
-  @Parameter(property = "jkube.kubernetesManifest", defaultValue = "${basedir}/target/classes/META-INF/jkube/openshift")
```

Comment:
I believe both the Javadoc and the `@Parameter` annotation were wrong. I took this from https://github.com/eclipse/jkube/pull/2714/files#diff-6290a43bd73acd81a35c9ece497c1315ed7e146c5e52a39887ccda6f40c29868R40.

Reply:
Yes, it's wrong